tf_files/configs/cluster.yaml

```diff
@@ -42,49 +64,91 @@ worker:
           name: ${cluster_name}
           managedPolicies:
             - arn: "arn:aws:iam::aws:policy/CloudWatchLogsFullAccess"
-            #- arn: "arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess"
-
-controller:
-  subnets:
-  - name: private
-  loadBalancer:
-    private: true
-
-etcd:
-  dataVolume:
-    encrypted: true
-  subnets:
-  - name: private
-
-routeTableId: ${route_table_id}
-# CIDR for Kubernetes VPC. If vpcId is specified, must match the CIDR of existing vpc.
-# vpcCIDR: "10.0.0.0/16"
-vpc:
-  id: ${vpc_id}
-
-# CIDR for Kubernetes subnet when placing nodes in a single availability zone (not highly-available) Leave commented out for multi availability zone setting and use the below `subnets` section instead.
-# instanceCIDR: "10.0.0.0/24"
+kubeDns:
+  autoscaler:
+    coresPerReplica: 256
+    nodesPerReplica: 16
+    min: 2
+kubeProxy:
+  ipvsMode:
+    enabled: false
+    scheduler: rr
+    syncPeriod: 300s
+    minSyncPeriod: 60s
+kubernetesDashboard:
+  adminPrivileges: true
+  insecureLogin: false
+addons:
+  clusterAutoscaler:
+    enabled: true
+  rescheduler:
+    enabled: false
+  metricsServer:
+    enabled: false
+useCalico: true
+kubelet:
+  RotateCerts:
+    enabled: true
```

Comment:
RotateCerts should be `false` - only applies when `tlsBootstrap` below is `true`, but I think `tlsBootstrap` is flaky ...

Reply:
changed